bluesky-tiled-plugins/bluesky_tiled_plugins/tiled_writer.py

```diff
@@ -467,6 +471,11 @@ def stream_datum(self, doc: StreamDatum):
 
     def datum(self, doc: Datum):
         doc = copy.copy(doc)
+        # Mark the Datum document with the spec of the corresponding Resource, if known
```

Comment:
What does this achieve? Is this a performance optimization to avoid doing a chained lookup Datum -> Resource -> Spec?